bin/dump_wiki

```diff
@@ -42,7 +44,11 @@ if (argv.h) {
     argParser.showHelp();
     process.exit(1);
 }
-
+if (!argv.dataBase && !argv.saveDir) {
```

Comment:
+1. Running a dump without storing the results is an important use case that we should continue to support.